tensorrt_llm/_torch/pyexecutor/connector.py

```
@@ -0,0 +1,415 @@
+        This is done to prevent the runtime from attempting to load the KV cache for these requests.
+
+        Args:
+            scheduled_requests: The scheduled requests.
+
+        Returns:
+            The scheduled requests with the context requests that are being loaded asynchronously removed.
+        """
+        allowed_context_requests = []
+
+        for req in scheduled_requests.context_requests:
+            # If this request is being loaded asynchronously, in addition to removing it from the list of scheduled requests,
```

Ruff (0.12.2)

Line too long (125 > 120)

(E501)